spec/support/view_component/example_group.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Support
+  module ViewComponent
+    module ExampleGroup
+      extend ActiveSupport::Concern
+
+      included do
+        subject(:rendered) { render_inline component }
+        let(:_test_context) { Context.new }
```

Comment:
Why are we putting `_` before the test name? Is it a new convention of some type?

Reply:
Good question. TestContext is an implementation detail of the test setup and as such I wanted to hide it as much as possible to avoid accidental override. `_` is a convention I encounter in javascript to denote private/public interface of the objects. Alternatively I could come up with some sort of registry that creates a new example-specific context before the test and then removes in after hook, but this seems a bit over the top :P

Reply:
fine, just wanted to make sure that it is our convention and not some Ruby / Rails magic :)